### ISSUE TITLE
ClickOnce app in 'Top Sites' launches app

### ISSUE THREAD

**Issue by whit0313** [brz/Breez-ClickOnce#1]
Love this extension - it works beautifully - but I am having one issue.

I have a ClickOnce app that I launch frequently, but it was launching unexpectedly at odd times while using Firefox. Eventually I realised that's because the ClickOnce app's URL was in the "Top Sites" section of the Firefox 'new tab' page. And every time I opened a new tab, Firefox attempted to create a preview for the URL of my ClickOnce app - which triggered the Breez-ClickOnce extension to launch the app.

I've fixed this for myself by simply removing my ClickOnce app's URL from the Top Sites section of the new tab page. But we have lots of users and I don't really want to have to explain to each of them how to do that.

Is it possible to alter this extension so that it only launches the ClickOnce app when the user clicks on a link? But not when the browser is trying to preview a page for (eg) the Top Sites section?

**Comment by brz** [comment]
Thank you for reporting this issue. I was not aware of the problem since I have disabled the "Top Websites" functionality entirely.
I was able to reproduce the problem and find a solution.
I will commit a fix later today, along with a new version of the extension.

**Comment by whit0313** [comment]
Brilliant! Many thanks.

Brendan.


--
Brendan White
sent from my iPhone

On 11 Jan 2018, at 5:56 pm, brz <notifications@github.com<mailto:notifications@github.com>> wrote:


Thank you for reporting this issue. I was not aware of the problem since I have disabled the "Top Websites" functionality entirely.
I was able to reproduce the problem and find a solution.
I will commit a fix later today, along with a new version of the extension.

**Comment by brz** [comment]
Version 1.2 of the extension has been published in the firefox addons gallery. Can you check if this solves your problem so I can close the issue?

**Comment by whit0313** [comment]
Thanks! I’ll be at work in about four hours, I’ll try it when I get there.


--
Brendan White
sent from my iPhone

On 11 Jan 2018, at 5:56 pm, brz <notifications@github.com<mailto:notifications@github.com>> wrote:


Thank you for reporting this issue. I was not aware of the problem since I have disabled the "Top Websites" functionality entirely.
I was able to reproduce the problem and find a solution.
I will commit a fix later today, along with a new version of the extension.
